src/haz3lcore/zipper/Zipper.re

```diff
@@ -246,7 +246,20 @@ let rec construct = (from: Direction.t, label: Label.t, z: t): IdGen.t(t) => {
       /* Special case for constructing string literals.
          See Insert.move_into_if_stringlit for more special-casing. */
       construct(Left, [Form.string_delim ++ Form.string_delim], z)
+    | [content] when Form.is_comment(content) =>
```

Comment:
more reason to distinguish proper whitespace from other secondary notational things